site/.gitignore

```diff
@@ -1 +1,3 @@
 *~
+/site-deployment.yaml
```

Comment:
ignored because they are created dynamically from yaml.in?

Reply:
Yes.

I think a general rule that should be "git status" should still be clean after building the project, so all derived objects should be ignored.